packages/react-instantsearch/examples/autocomplete/README.md

```diff
@@ -0,0 +1,8 @@
+This example shows how to build an autocomplete with react-instantsearch
+
+To start the example: 
+
+* npm install
+* npm start
+
+Read more about react-instantsearch [in our documentation](https://community.algolia.com/instantsearch.js/react/).
```

Comment:
Thanks to @zasmail that tried the example yesterday, there's one part that is complex to understand: the namespacing when gathering all the data. I will detail this using comment in the code. I'll also add details in the README to explain the main idea behind the example, but without going to far as it might heavily depends on the underlying autocomplete component you choose to use.